test/examples/TrafficLight.test.js

```diff
@@ -0,0 +1,24 @@
+var assert = require('assert');
+describe("Traffic Light", function() {
+    var include = require('../helpers/Utils').include;
+    include('Core.js');
+    Core.registerEventPoint('DOM_Init'     , {log: false});
+    include('examples/states/trafficLight/Car.js');
+    include('examples/states/trafficLight/TrafficLight.js');
+
```

Comment:
How to create correct tests cases for functional tests?

I will explain using this example. You have entities: a traffic light and cars.
Try to compose names of tests cases of functional tests the way that will explain how these entities use each other.

In this example cars use a traffic light to know when to move and when to start. So functional tests for a traffic light will be based on cases of cars usage of it.